modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/tests

```diff
@@ -29,6 +39,17 @@
     requirement = 'The system shall be able to solve two-phase flow in a channel using the mixture drift flux model.'
     installation_type = in_tree
   []
+  [mixture_channel_drift_flux_physics]
+    type = 'Exodiff'
+    input = 'channel-drift-flux-physics.i'
+    exodiff = 'channel-drift-flux-physics_out.e'
+    requirement = 'The system shall be able to solve two-phase flow in a channel using a mixture drift flux model using Navier Stokes physics syntax.'
+    installation_type = in_tree
+    # Negative viscosity during solve
+    allow_warnings = true
+    custom_cmp = 'mixture_physics_diff.cmp'
+    min_ad_size = 80
```

Comment:
we need to figure out why your physics test needs this restriction while the non-physics test doesn't

Reply:
with the fixes, both tests now require the min ad size. it was really random that only the physics test did